NavigationViewExample/Transitions/DismissalAnimator.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+import MapboxMaps
+import MapboxNavigation
+
+class DismissalAnimator: NSObject, UIViewControllerAnimatedTransitioning {
```

Comment:
`Presentation` and `DismissalAnimator` are implemented by the client. Such example will be added also to examples repo in scope of https://github.com/mapbox/mapbox-navigation-ios-examples/issues/170.